.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+**Change Description**
```

Comment:
can we put check lists in these PR templates like:

- [ ] BUG
- [ ] User Experience
- [x] RPC

alternatively we can use tags, but it would be nice to have the template provide some very basic set of things a developer could click to indicate the areas-affected/scope of the change

Reply:
Sure, the template is just markdown, so we can put checkboxes wherever.  Do you have a preference between keeping the template small vs letting it get larger?  For example, we could add just a few checkboxes at the top or we can make this thing more like a form, where we provide checkboxes for what aspects are impacted (like documentation, APIs, consensus, specific plugins, etc.) and then ask for comments about each box checked.

Reply:
willing to entertain opinions on it.  My gut is telling me that its more likely to include information if inputing said information is trivial.  Maybe some amount of "form" like stuff is warranted?